libpod/container_api.go

```diff
@@ -218,48 +220,28 @@ func (c *Container) Kill(signal uint) error {
 
 // Exec starts a new process inside the container
 // TODO investigate allowing exec without attaching
```

Comment:
Can you update the comment here? Particularly to explain the returns - it's not immediately obvious the int is an exit code